src/lib/utils/http-request.ts

```diff
@@ -37,12 +37,23 @@ export const httpRequest = async (
 
   const endpoint = `${baseUrl}${path}`;
 
+  const urlParams = new URLSearchParams();
+  for (const [key, value] of Object.entries(requestParams)) {
```

Comment:
Maybe we would also like to make the "any" type of parameter values more strict in the future 😢